.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: File a bug report
+title: Describe the problem
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Your issue will be triaged by the RNW team according to this process: https://github.com/microsoft/react-native-windows/wiki/Triage-Process
+-->
+## Environment
+Run the following in your terminal and copy the results here.
+1. `react-native -v`:
+2. `react-native run-windows --info`:
+3. `reg query "HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock"`
+
+<!-- Consider including this information as well:
+What SDK version are you building for? Choose from 10.0.15063, 10.0.16299, 10.0.18362, etc.
+- Target Platform Version(s):
+What device(s) are you targeting? Choose any from Desktop, Xbox, Hololens)
+- Target Device(s):
+Which version of Visual Studio are you using? Specify Visual Studio 2017 or Visual Studio 2019
+- Visual Studio Version:
+Which build configuration are you running? Choose from Debug, DebugBundle, Release, ReleaseBundle 
+- Build Configuration:
+-->
```

Comment:
I took this out of the default shown text. How much can we incorporate into `info` and leave out entirely?

Reply:
SDKS installed is already represented with 'react-native run-windows --info':
```
  System:
    OS: Windows 10 10.0.18363
    CPU: (8) x64 Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz
    Memory: 5.62 GB / 15.92 GB
  Binaries:
    Node: 12.10.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.17.3 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.10.3 - C:\Program Files\nodejs\npm.CMD
  npmPackages:
    react: 16.11.0 => 16.11.0
    react-native: ^0.62.0-0 => 0.62.2
    react-native-windows: ^0.62.0-0 => 0.62.2
  Installed UWP SDKs:
    10.0.14393.0
    10.0.17763.0
    10.0.18362.0
```
Note that seems to be the only information not also reported with just `react-native info` (other than the RNW version, which is a pretty important bit):
```
info Fetching system and libraries information...
System:
    OS: Windows 10 10.0.18363
    CPU: (8) x64 Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz
    Memory: 5.71 GB / 15.92 GB
  Binaries:
    Node: 12.10.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.17.3 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.10.3 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
  IDEs:
    Android Studio: Not Found
  Languages:
    Java: Not Found
    Python: 2.7.16
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.11.0 => 16.11.0
    react-native: ^0.62.0-0 => 0.62.2
  npmGlobalPackages:
    *react-native*: Not Found
```
Only additional thing we're getting with `react-native -v` is the CLI version:
```
react-native-cli: 2.0.1
react-native: 0.62.2
```